lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -71,6 +71,10 @@ export class SearchFilterComponent implements OnInit {
         }
     }
 
+    get isFacetQueriesDefined() {
+        return !!(this.queryBuilder.config && this.queryBuilder.config.facetQueries);
```

Comment:
They still can be defined as `[]`, so I suggest checking the length as well

Reply:
should we make this accessible?

Reply:
Yes, I'm fine with making that public